includes/admin/class-wp-job-manager-admin-notices.php

```diff
@@ -317,16 +332,40 @@ private static function generate_notice_from_updates( $updates ) {
 	 * Renders a notice.
 	 *
 	 * @param array $notice See `generate_notice_from_updates` for format.
+	 * @param string $notice_id  Unique identifier for the notice.
 	 * @return void
 	 */
-	private static function render_notice( $notice ) {
-		// TODO Handle different levels of notices.
-		echo '<div class="notice wpjm-admin-notice wpjm-admin-notice--info">';
+	private static function render_notice( $notice_id, $notice ) {
+		if ( empty( $notice['actions'] ) || ! is_array( $notice['actions'] ) ) {
+			$notice['actions'] = [];
+		}
+
+		$notice_class = '';
+		if ( ! empty( $notice['style'] ) ) {
+			$notice_class = 'wpjm-notice-' . $notice['style'];
+		}
+
+		$is_dismissible       = $notice['dismissible'];
+		$notice_wrapper_extra = '';
+		if ( $is_dismissible ) {
+			wp_enqueue_script( 'job_manager_notice_dismiss' );
+			$notice_class        .= ' is-dismissible';
+			$notice_wrapper_extra = sprintf( ' data-dismiss-action="wpjm_dismiss_notice" data-dismiss-notice="%1$s" data-dismiss-nonce="%2$s"', esc_attr( $notice_id ), esc_attr( wp_create_nonce( self::DISMISS_NOTICE_NONCE_ACTION ) ) );
+		}
+
+		// TODO Remove hard-coded 'is-dismissable' CSS class once all notices are converted to use this class.
+		echo '<div class="notice wpjm-admin-notice wpjm-admin-notice--info is-dismissible' . esc_attr( $notice_class ) . '"' . esc_attr__( $notice_wrapper_extra ). '>';
 
 		echo '<div class="wpjm-admin-notice__top">';
-		// TODO Add icon.
+
+		// TODO Implement get_icon method.
+		if ( ! empty( $notice['icon'] ) ) {
+			// phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped -- Dynamic parts escaped in the function.
+			echo get_icon( $notice['icon'], 'wpjm-notice__icon' );
+		}
+
 		echo '<div class="wpjm-admin-notice__message">';
-		echo wp_kses( $notice['message'], self::ALLOWED_HTML );
+		echo '<strong>' . wp_kses( $notice['message'], self::ALLOWED_HTML ) . '</strong>';
```

Comment:
maybe we want to add `font-weight: bold` to the `wpjm-admin-notice__message` better?

Reply:
I had gone with `<strong>` for the semantic significance (i.e. this is an important notice for screen readers). Maybe that's a misplaced concern, and if so, I can switch this out to the `font-weight` property.

Or, is there another consideration between the two that I'm leaving out?

Reply:
🤔 I didn't thought about the semantic significance... makes sense to me now :)